src/fu-util-common.c

```diff
@@ -1761,18 +1803,26 @@ fu_util_release_to_string(FwupdRelease *rel, guint idt)
 					   fwupd_release_get_update_message(rel));
 	}
 
+	/* TRANSLATORS: release attributes */
+	title = _("Release Flags");
 	for (guint i = 0; i < 64; i++) {
 		if ((flags & ((guint64)1 << i)) == 0)
 			continue;
-		g_string_append_printf(flags_str,
-				       "%s|",
-				       fwupd_release_flag_to_string((guint64)1 << i));
-	}
-	if (flags_str->len > 0) {
-		g_string_truncate(flags_str, flags_str->len - 1);
-		/* TRANSLATORS: release properties */
-		fu_common_string_append_kv(str, idt + 1, _("Flags"), flags_str->str);
+		tmp2 = fu_util_release_flag_to_string((guint64)1 << i);
+		if (tmp2 == NULL)
+			continue;
+		if (title != NULL) {
+			g_autofree gchar *bullet = NULL;
+			bullet = g_strdup_printf("• %s", tmp2);
+			fu_common_string_append_kv(str, idt + 1, title, bullet);
+			title = NULL;
+		} else {
+			g_autofree gchar *bullet = NULL;
+			bullet = g_strdup_printf("• %s", tmp2);
+			fu_common_string_append_kv(str, idt + 1, "", bullet);
+		}
```

Comment:
How about `title != NULL ? title : ""`?

Reply:
Yes, and we can make it one line simpler too. Thanks.